docs/03consensus/attacks_and_countermeasures.md

```diff
@@ -5,45 +5,79 @@ sidebar_position: 14
 # 3.14 Relevant Attacks and Countermeasures
 
 ## 51% attack
-A 51% (actually 46%, as explained below) attack involves creating an alternate chain which eventually reaches a higher weight than the honest chain, and forces users to re-org. This is the classic long-range attack, which is present on many blockchain networks including Chia's, as well as on proof-of-work systems.
-
-In this attack, someone who controls at least 46% of the network space creates an alternate chain, which eventually becomes heavier than the honest chain.
+A 51% (actually 42.6%, as explained below) attack involves creating an alternate chain which eventually reaches a higher weight than the honest chain, and forces users to re-org. This is the classic long-range attack, which is present on many blockchain networks including Chia's, as well as on Proof-of-Work systems.
 
 There are two main differences between Chia's Proof of Space and Time consensus and Proof of Work.
-1. In Chia, the attacker can extend and farm on many chains simultaneously, making the attack possible with only 46% of the network space. A fast VDF is not required to gain this advantage. Therefore, a 46% assumption should used as the base line for a long-range attack.
-2. In Chia, if the attacker has the fastest VDF on the entire network, they can get an additional space advantage/boost. (If the attacker's VDF is even slightly slower than the fastest VDF, it will not give them any advantage.)
+1. In Chia, an attacker can extend and farm on many chains simultaneously, making the attack possible with only 42.6% of the network space. A fast timelord is not required to gain this advantage. Therefore, a 42.6% assumption should used as the base line for a long-range attack.
```

Comment:
need to have the fastest timelord (tied with the honest network) to perform this attack